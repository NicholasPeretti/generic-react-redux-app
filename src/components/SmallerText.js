import React from 'react'
import Text from './Text'

const style = {
  fontSize: '12px'
}

const SmallerText = ({ children }) => (
  <Text style={style}>
    {children}
  </Text>
)

export default SmallerText
