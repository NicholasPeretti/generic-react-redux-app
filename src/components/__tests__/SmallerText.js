import React from 'react'
import { shallow } from 'enzyme'
import PresentationalTest from '../../utils/tests/presentational'
import SmallerText from '../SmallerText'

describe('SmallerText', () => {
  PresentationalTest(SmallerText)

  test('Should match snapshot', () => {
    const wrapper = shallow(<SmallerText>This is a SmallerText</SmallerText>)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
