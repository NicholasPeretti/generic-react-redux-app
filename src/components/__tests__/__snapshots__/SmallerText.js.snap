// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SmallerText Should match snapshot 1`] = `"<span class=\\"sc-bdVaJa sc-bwzfXH cKrwmx\\">This is a SmallerText</span>"`;
